spec/cancan/model_adapters/active_record_adapter_spec.rb

```diff
@@ -237,101 +237,84 @@ class User < ActiveRecord::Base
     end
 
     it 'has false conditions if no abilities match' do
-      expect(@ability.model_adapter(Article, :read).conditions).to eq("'t'='f'")
+      expect(@ability.model_adapter(Article, :read).database_records.to_sql.strip.squeeze(' '))
```

Comment:
the previous test should still be in place, right? with `expect(@ability.model_adapter(Article, :read).conditions).to eq("1 == 0")`